openedx/core/djangoapps/user_api/accounts/permissions.py

```diff
@@ -0,0 +1,20 @@
+"""
+Permissions classes for User accounts API views.
+"""
+from __future__ import unicode_literals
+
+from rest_framework import permissions
+
+
+class IsSuperUserOrCanDeactivateUser(permissions.BasePermission):
```

Comment:
There is no need to reference superuser in the name of this class or its implementation. Superusers have all permissions by default. This should simply be `CanDeactivateUser`.